R/geom_phylopic.R

```diff
@@ -182,6 +189,109 @@ GeomPhylopic <- ggproto("GeomPhylopic", Geom,
   }
 )
 
+phylopic_env$glyph_ind <- 1
+
+#' Use PhyloPics as legend key glyphs
+#'
+#' Specify existing images, taxonomic names, or PhyloPic uuids to use PhyloPic
+#' silhouettes as legend key glyphs in a ggplot plot.
+#'
+#' @param img A [Picture][grImport2::Picture-class] or png array object, e.g.,
+#'   from using [get_phylopic()].
+#' @param name \code{character}. A taxonomic name to be passed to [get_uuid()].
+#' @param uuid \code{character}. A valid uuid for a PhyloPic silhouette (such as
+#'   that returned by [get_uuid()] or [pick_phylopic()]).
+#' @details One (and only one) of `img`, `name`, or `uuid` must be specified.
+#'
+#'   This argument may also be a vector/list of valid values if multiple
+#'   silhouettes should be used as key glyphs. In this case, the specified
+#'   silhouettes will be used as ordered as key glyphs one by one, with
+#'   recycling as necessary.
+#'
+#'   Note that the relative sizes of the silhouettes are fixed. Therefore, this
```

Comment:
No straightforward way of addressing this?

Reply:
I'll give it another go, but the last time I tried it made the silhouettes REALLY big.

Reply:
OK, have a little bit of a play, but don't sweat over it for now. Colour/fill is likely to be the main aesthetics we need to get right.